Arrowgene.Ddon.GameServer/Characters/EquipManager.cs

```diff
@@ -10,6 +10,11 @@ namespace Arrowgene.Ddon.GameServer.Characters
 {
     public class EquipManager
     {
+        public static EquipType GetEquipTypeFromSlotNo(ushort slotNo)
+        {
+            return (slotNo > 15) ? EquipType.Visual : EquipType.Performance;
```

Comment:
This isn't correct. In the pawn equipment storage, slot 31 would be pawn 2's performance equipment. You could also add a new method to the Equipment class (the wrapper for the equipment Storage instances), to get the equip type and equipment slot from a storage slot. That way you could do something like common.Equipment.GetEquipTypeAndEquipSlotFromStorageSlot(slotNo) for example.

Reply:
I'm pretty sure it is correct. I also adjusted the pawn offsets before passing it into this function. If you look, it is called relative offset. I suppose I could swap the logic and make it do it internally instead.

Reply:
nevermind, i just saw the updated commit